pyproject.toml

```diff
@@ -87,3 +85,10 @@ exclude = '''
 packages = ["rst2pdf"]
 
 [tool.setuptools_scm]
+
+[tool.uv]
+dev-dependencies = [
+    "pymupdf~=1.0,<1.24.11",
```

Comment:
Note there's something funky going on in `pymupdf==1.24.11` as per https://github.com/pymupdf/PyMuPDF/issues/3931. From what it sounds like it's out of their control, but it does make the test results flood with warnings that we can't fix.

Opted to use lower versions to avoid the noise.

Reply:
Thank you! I think that's a good decision.